website/static/js/institutionProjectSettings.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+var $ = require('jquery');
+var ko = require('knockout');
+var Raven = require('raven-js');
+
+var $osf = require('js/osfHelpers');
+var ctx = window.contextVars;
```

Comment:
`contextVars` should not be used in re-usable modules because it makes them less..re-usable. Instead, pass any necessary variables to the constructor of the wrapper class and view-model.